src/select/SimpleSelect.js

```diff
@@ -668,65 +648,73 @@ class SimpleSelect extends Component {
+        <p id="aria-selection-event">{ariaLiveSelection}</p>
+        <p id="aria-context">{this.constructAriaLiveMessage()}</p>
       </div>
     )
   }
 
   renderInput() {
+    const {
+      'aria-label': ariaLabel,
+      'aria-labelledby': ariaLabelledBy,
+      disabled,
+      inputComponent,
+      inputId,
+      inputValue,
+      showInput
+    } = this.props
+
     const inputProps = {
-      'aria-label': this.props['aria-label'],
-      'aria-labelledby': this.props['aria-labelledby'],
-      disabled: this.props.disabled,
-      getRef: (ref) => { this.input = ref },
-      id: this.props.inputId,
-      inputValue: this.props.inputValue,
+      'aria-label': ariaLabel,
+      'aria-labelledby': ariaLabelledBy,
+      disabled,
+      getRef: (ref) => {
+        this.input = ref
+      },
+      id: inputId,
+      inputValue,
       onBlur: this.onInputBlur,
       onChange: this.onInputChange,
       onFocus: this.onInputFocus
     }
-    const InputComponent = this.props.inputComponent
+    const InputComponent = inputComponent
 
-    return this.props.showInput
-      ? <InputComponent {...inputProps} />
-      : this.renderFocusPlaceholder(inputProps)
+    return showInput ? <InputComponent {...inputProps} /> : this.renderFocusPlaceholder(inputProps)
   }
 
   renderFocusPlaceholder(inputProps) {
-    const FocusPlaceholderComponent = this.props.focusPlaceholderComponent
-    const {
-      onChange,
-      inputValue,
-      ...placeholderProps
-    } = inputProps
+    const { focusPlaceholderComponent } = this.props
+    const FocusPlaceholderComponent = focusPlaceholderComponent
+    const { onChange, inputValue, ...placeholderProps } = inputProps
     return <FocusPlaceholderComponent {...placeholderProps} />
   }
 
   render() {
-    const {
-      beforeInput,
-      disabled,
-      id,
-      showValuesWhileFocused,
-      valueGroupComponent
-    } = this.props
+    const { beforeInput, className, disabled, id, showValuesWhileFocused, valueGroupComponent } =
+      this.props
     const { isFocused, isOuterFocused } = this.state
+    // eslint-disable-next-line react/destructuring-assignment
     const isOpen = this.props.isOpen || this.state.isOpen
-    const {
-      valueGroupComponent: _valueGroupComponent,
-      ...valueGroupProps
-    } = this.props
+    const { valueGroupComponent: _valueGroupComponent, ...valueGroupProps } = this.props
     const ValueGroupComponent = valueGroupComponent
-    const selectClassName = classNames('crane-select', this.props.className, {
-      open: isOpen, focus: isFocused, 'outer-focus': isOuterFocused, disabled
+    const selectClassName = classNames('crane-select', className, {
+      open: isOpen,
+      focus: isFocused,
+      'outer-focus': isOuterFocused,
+      disabled
     })
 
     return (
-      <div aria-expanded={isOpen} className={selectClassName} ref={(container) => { this.container = container }}>
```

Comment:
Hmm why remove this ref?

Reply:
Eslint flagged it and I looked and could not find any usages.